packages/palette_generator/test/encoded_images.dart

```diff
@@ -0,0 +1,1351 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:typed_data';
+
+// This file contains base64-ecoded versions of the images in assets/, so that
+// unit tests can be run on web where loading from files isn't an option and
+// loading from assets requires user action.
+
+/// Data version of assets/dominant.png.
+final Uint8List kImageDataDominant = base64.decode(
+    'iVBORw0KGgoAAAANSUhEUgAAAGQAAABkCAIAAAD/gAIDAAAACXBIWXMAAAsTAAALEwEAmpwYAA'
+    'AAqElEQVR42u3WoQ0AIAxFwV/C/ivDBmAgIO7ZpuZS0RpZVetx7i3/WItgwYIFCxYswYL1uF7Z'
+    '/PCMXBYsWLBgwRIsWLD+qZJBwWXBggULFizBggULFixYggULFixYsAQLFixYsGAJFixYsGDBEi'
+    'xYsGDBgiVYsGDBggVLsGDBggULlmDBggULFizBggULFixYggULFixYsAQLFixYsGAJFixYsGDB'
+    'EqzzTawXBchMuogIAAAAAElFTkSuQmCC');
+
+/// Data version of assets/tall_blue.png.
+final Uint8List kImageDataTallBlue = base64.decode(
+    'iVBORw0KGgoAAAANSUhEUgAAAAEAAAPoCAIAAACQ1AMJAAAACXBIWXMAAAsTAAALEwEAmpwYAA'
+    'AAH0lEQVRIx+3DQQkAAAgAscP+nTWGnw1W7VSqqqrq3wOa1wjO6jYVpgAAAABJRU5ErkJggg='
+    '=');
+
+/// Data version of assets/wide_red.png.
+final Uint8List kImageDataWideRed = base64.decode(
+    'iVBORw0KGgoAAAANSUhEUgAAA+gAAAABCAIAAADCYhNkAAAACXBIWXMAAAsTAAALEwEAmpwYAA'
+    'AAG0lEQVRIx+3BMQEAAAwCoNk/9KzhAeQPAABYV8RfAQE8QBqiAAAAAElFTkSuQmCC');
```

Comment:
All the images look good to me (visually), verified with this:

https://jsfiddle.net/ditman/fb8qLmku/